playground/AzureStorageEndToEnd/AzureStorageEndToEnd.AppHost/Program.cs

```diff
@@ -13,12 +13,14 @@
        .WithExternalHttpEndpoints()
        .WithReference(blobs);
 
+#if !SKIP_DASHBOARD_REFERENCE
```

Comment:
NIT: Should the comment bellow be updated to say "pass /p:SkipDashboardProjectReference=true" instead of "Comment out this code"?

Reply:
done.